vmd-server/src/server/run.rs

```diff
@@ -0,0 +1,47 @@
+// === External crates ========================================================
+
+use hyper::{
+    Server as HyperServer,
+    server::conn::AddrIncoming,
+};
+use swagger::EmptyContext;
+use std::net::ToSocketAddrs;
+use log::info;
+
+// === Internal modules =======================================================
+
+use vmd_rust_server_api::{
+    server::MakeService,
+    server::context::MakeAddContext,
+};
+
+use super::{
+    util::VmdResult,
+    api::ApiImpl,
+    tls::VmdTlsAcceptor,
+    cli::Args,
+};
+
+// === Implementations ========================================================
+
+pub(crate) async fn run(args: &Args) -> VmdResult<()>
+{
+    println!("⚡    Virtual Machine Daemon Server   ⚡\n");
```

Comment:
`vmd` does not actually come from those words

Reply:
Oh? What is the abbreviation then?

Reply:
From an inside joke.

Reply:
I propose virtual machine daemon as disambiguation :P

Reply:
https://github.com/tiiuae/vmd/pull/35/commits/d9673e712f08f5fd4718c4b1585d53e5f9282024